### PR TITLE
Translation (and review) of docs/practice section

### PR DESCRIPTION
**Pull request by dxps**
This is my translation of docs/practice section.
An additional review of a native chinese speaker may help.

### CODE REVIEW

**Comment by kvetoslavnovak** [comment]
Generally looks fine to me.

Some issues: 

**Not Translated pages**
- http://beta.ant.design/changelog - heading translation should be enough
http://beta.ant.design/docs/react/upgrade-notes - - heading translation should be enough


**Not Fully Translated Components**
- http://beta.ant.design/components/mention/


**Localisation**
- there are components ( Pagination, DatePicker, TimePicker, Calendar, Popconfirm, Table, Modal, Button, Select, Transfer, Radio) which should use LocalProvider en_US to render according to http://beta.ant.design/components/locale-provider/ 


**In Code Text**
Code examples still contain some Chinese texts. Here is my detail list. I do not have time for all changes so everyone is welcomed to help:
- http://beta.ant.design/components/button/ - Icon example
- http://beta.ant.design/components/checkbox/ - Check all example
- http://beta.ant.design/components/cascader/ - all cascading texts
- http://beta.ant.design/components/input/ - Basic usage example - placeholder text
- http://beta.ant.design/components/radio/ - Radio style and Radio size  examples
- http://beta.ant.design/components/select/ - Automatic completion and Coordinate examples
- http://beta.ant.design/components/slider/ -Graduated slider example
- http://beta.ant.design/components/upload/ - Uploading by clicking example
- http://beta.ant.design/components/alert/ - Icon example
- http://beta.ant.design/components/badge/ - Dynamic example
- http://beta.ant.design/components/dropdown/ - all examples
- http://beta.ant.design/components/modal/ - Basic example
- http://beta.ant.design/components/popconfirm/ - Basic, Placement, Conditional trigger examples
- http://beta.ant.design/components/popover/ - all examples
- http://beta.ant.design/components/progress/ - Custom text format example
- http://beta.ant.design/components/table/ - dataSource in all examples
- http://beta.ant.design/components/timeline/ - all examples
- http://beta.ant.design/components/tooltip/ - Tooltip button example
- http://beta.ant.design/components/menu/ - all examples
- http://beta.ant.design/components/steps/ - all examples
- http://beta.ant.design/components/tabs/ - Position example
- http://beta.ant.design/components/back-top/ - all examples
- http://beta.ant.design/components/spin/ - Embedded mode and Customize description examples

**Comment by benjycui** [comment]
@kvetoslavnovak trace #1471

File: docs/practice/cases.en-US.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+order: 0
+title: Cases
+---
+
+Ant Design is an implementation of UI Design Language。
```
Comment (benjycui) [inline docs/practice/cases.en-US.md]: Ant Design is one design language for enterprise-like complex UIs.

File: docs/practice/cases.en-US.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Starting in April 2015, Ant Design product rapidly expanded, covering multiple business lines and more than 40 systems.
```
Comment (benjycui) [inline docs/practice/cases.en-US.md]: Starting in April 2015, more and more products of Ant Financial follow Ant Design specification, ....

File: docs/practice/cases.en-US.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Based in Taiwan, used by both professional and non-professional designers,
```
Comment (benjycui) [inline docs/practice/cases.en-US.md]: Designed for enterprise-like complex UIs, ...

**Comment by benjycui** [comment]
@visvadw hope that my comments will help you to understand these documentation, thanks!

**Comment by benjycui** [comment]
@afc163 产品名就这样翻译了么。。我看了那些网站，好像没有英文名。

File: docs/practice/cases.en-US.md (hunk 1 above)
Comment (afc163) [inline docs/practice/cases.en-US.md]: `a` is ok, this is not slogan.

**Comment by afc163** [comment]
产品名翻译的都没啥问题。

**Comment by visvadw** [comment]
@benjycui Thanks for the corrections. 👍 I was sure that some parts were not correctly translated.
@afc163 Thanks also.

**Comment by kvetoslavnovak** [comment]
I have updated the issues list.

**Pull request merged**